src/doc/rustc/src/SUMMARY.md

```diff
@@ -27,6 +27,7 @@
     - [armv7-unknown-linux-uclibceabihf](platform-support/armv7-unknown-linux-uclibceabihf.md)
     - [\*-android and \*-androideabi](platform-support/android.md)
     - [\*-linux-ohos](platform-support/openharmony.md)
+    - [\*-esp-esp-idf](platform-support/esp-idf.md)
```

Comment:
```suggestion
    - [\*-esp-idf](platform-support/esp-idf.md)
```

Reply:
This is not quite correct, I've changed it to `-esp-espidf`.